appveyor.yml

```diff
@@ -64,11 +70,17 @@ install:
     - cmd: rmdir C:\cygwin /s /q
 
     # Use the pre-installed Miniconda for the desired arch
-    - ps: if($env:TARGET_ARCH -eq 'x86')
-            {$root = "C:\Miniconda"}
-          else
-            {$root = "C:\Miniconda-x64"}
-          $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    #
+    # However, it is really old. So, we need to update some
```

Comment:
You might be able to call conda with a absolute path here and not mess with path.  Not 100% sure, but it might work.